flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/ZooKeeperCompletedCheckpointStoreTest.java

```diff
@@ -58,6 +66,54 @@ public void testPathConversion() {
 		assertEquals(checkpointId, ZooKeeperCompletedCheckpointStore.pathToCheckpointId(path));
 	}
 
+	private static class TestException extends RuntimeException{
```

Comment:
Could we reuse `ExpectedTestException` from `runtime` `testutils` package?

Reply:
I searched for suitable `TestException` and missed `ExpectedTestException` :) 
Will use it, thanks!